src/library_syscall.js

```diff
@@ -864,11 +864,6 @@ var SyscallsLibrary = {
     FS.rename(oldpath, newpath);
     return 0;
   },
-  __syscall_linkat__nothrow: true,
-  __syscall_linkat__proxy: false,
-  __syscall_linkat: function(olddirfd, oldpath, newdirfd, newpath, flags) {
-    return -{{{ cDefs.EMLINK }}}; // no hardlinks for us
-  },
```

Comment:
I thought this PR removes link and switches to linkat, but this diff removes linkat?

Reply:
__syscall_linkat not lives in `emscripten_syscall_stubs.c` where all the other stubs live (and where `__syscall_link` previously lived).